Source/WebCore/dom/Document.cpp

```diff
@@ -8582,6 +8581,16 @@ CheckedRef<const FullscreenManager> Document::checkedFullscreenManager() const
 {
     return fullscreenManager();
 }
+
+Element* Document::fullscreenElement() const
+{
+    for (Ref element : makeReversedRange(topLayerElements())) {
+        if (element->hasFullscreenFlag())
+            return element.ptr();
+    }
+
+    return nullptr;
+}
```

Comment:
Any reason to move this to Document ? Can't we just leave this on FullscreenManager and access it from there? I'd prefer fullscreen logic to be contained.